frontend/registration/register.tsx

```diff
@@ -54,18 +54,23 @@ class _RegisterComponent extends React.PureComponent<Props, State> {
             </div>
         } else if (!this.state.hasConsented) {
             return <div>
-                <p>Step 1 of 2</p>
-                <h1>Consent</h1>
-                <p>Apocalypse Made Easy! is part of a study...</p>
+                <p className="steps">Step 1 of 2</p>
+                <h1>Consent #1</h1>
+                <p>Apocalypse Made Easy! is part of a study led by Dr. John Oliffe (email: john.oliffe@ubc.ca; phone: 604 822 7638), along with Andrew Munroe at the University of British Columbia, Melbourne-based First Person Consulting, and the Propel Centre for Population Health Impact at the University of Waterloo. As a Social Innovators Challenge Project funded by the Movember Foundation, its focus is on social connectedness.</p>
+                
+                <p>In order to study social connectedness and improve the program, the way you interact with Apocalypse Made Easy!, along with limited demographic data you supply while registering a profile, is recorded and stored. This data will be anonymized and stored on secure servers in Toronto, Ontario, and only accessible to the research team.</p>
```

Comment:
I think we should just say "in Canada" instead of "in Toronto, Ontario". Conceivably, the data could be backed up to BC or hosted at UBC at some point, etc.

Reply:
Yeah, good point.